instrumentation/apache-dubbo-2.7/javaagent/build.gradle.kts

```diff
@@ -10,33 +10,17 @@ muzzle {
     assertInverse.set(true)
   }
 }
+// The testing module use `class DubboBootstrap` which is since 2.7.5, the ut will fail for 2.7.0
+// so we only test it on (2.7.5,)
+val apacheDubboVersion = "2.7.5"
 
 dependencies {
   implementation(project(":instrumentation:apache-dubbo-2.7:library-autoconfigure"))
 
-  library("org.apache.dubbo:dubbo:2.7.0")
-}
-
-val latestDepTest = findProperty("testLatestDeps") as Boolean
+  library("org.apache.dubbo:dubbo:$apacheDubboVersion")
 
-testing {
-  suites {
-    // using a test suite to ensure that project(":instrumentation:apache-dubbo-2.7:library-autoconfigure")
-    // is not available on test runtime class path, otherwise instrumentation from library-autoconfigure
-    // module would be used instead of the javaagent instrumentation that we want to test
-    val testDubbo by registering(JvmTestSuite::class) {
-      dependencies {
-        implementation(project(":instrumentation:apache-dubbo-2.7:testing"))
-        if (latestDepTest) {
-          implementation("org.apache.dubbo:dubbo:+")
-          implementation("org.apache.dubbo:dubbo-config-api:+")
-        } else {
-          implementation("org.apache.dubbo:dubbo:2.7.0")
-          implementation("org.apache.dubbo:dubbo-config-api:2.7.0")
-        }
-      }
-    }
-  }
```

Comment:
could you explain why you decided to remove this

Reply:
From the comments,  module apache-dubbo-2.7/javaagent and  apache-dubbo-2.7/library-autoconfigure have the same unit class name. Some override will happen .

I have change the class name different, So those test cases work fine .

So I think the hack in gradle  do not need any more just like other module.

Reply:
This didn't have anything to do with the test class name. The problem is that when the `library-autoconfigure` module is on class path the tests pass even when the javaagent instrumentation does not work. You can test it by disabling the javaagent instrumentation (make the `defaultEnabled` method return `false`).

Reply:
Oh, the test pass when i disabled the javaagent instrumentation. 
I will revert the `kts` file